normandy/classifier/tests/test_api.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from normandy.base.utils import Whatever
+from normandy.recipes.tests import RecipeActionFactory, RecipeFactory
+
+@pytest.mark.django_db
+class TestFetchBundleAPI(object):
+
+    def test_it_works(self, client):
+        res = client.post('/api/v1/fetch_bundle/')
+        assert res.status_code == 200
+        assert res.data == {'recipes': []}
+
+    def test_it_serves_recipes(self, client):
+        recipe_action = RecipeActionFactory(arguments={'foo': 'bar'})
```

Comment:
Huh, I swear it used to used the [build strategy](http://factoryboy.readthedocs.org/en/latest/reference.html#factory.BUILD_STRATEGY) by default. 

What I was trying to convey was a suggestion to call `RecipeActionFactory.create` instead of just `RecipeActionFactory`, so that it's obvious whether you're creating it in the database or not regardless of what you set the default strategy to be.